### PR TITLE
Automated Test - COMMERCE-5907 | master

### PR DESCRIPTION
**Pull request by fmarcio**

### CODE REVIEW

**Comment by fmarcio** [comment]
https://issues.liferay.com/browse/COMMERCE-5907; cc: @lorenzolapescara

**Comment by pedro-michel** [comment]
ci:test:sf

**Comment by pedro-michel** [comment]
ci:test:sf

**Comment by lorenzolapescara** [comment]
Review started

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase [hunk 1]
```diff
@@ -46,12 +46,34 @@ definition {
...
+			locator1 = "CommerceAccelerators#ADD_TO_CART_BUTTON");
+
+		AssertElementNotPresent(
+			key_optionValue = "Break Fluid",
+			key_productName = "Break Fluid",
+			locator1 = "CommerceAccelerators#PRODUCT_LABEL_PRODUCT_AVAILABLE");
+
+		AssertElementNotPresent(
+			locator1 = "CommerceFrontStore#FS_PRODUCT_SKU",
+			value1 = "MIN93016A");
```
[the remark this answers is not in the document]
Comment (fmarcio) [reply modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase]: All requests done @lorenzolapescara

**Comment by pedro-michel** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-product-test/src/testFunctional/paths/CommerceEntry.path [hunk 2]
```diff
@@ -526,6 +526,11 @@
 	<td>//div[contains(@class,'card d-flex')]//h4[contains(.,'Option')]/..//a</td>
 	<td></td>
 </tr>
+<tr>
+	<td>PRODUCT_CARD_SKU</td>
+	<td>//div[@class="cp-information"]//span[contains(text(),'$key_productSKU')]//following::p[contains(.,'$key_productName')]</td>
+	<td></td>
+</tr>
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/paths/CommerceEntry.path:533]: Change requested to this locator in #960

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]

File: modules/apps/commerce/commerce-product-test/src/testFunctional/paths/CommerceEntry.path (hunk 2 above)
Comment (fmarcio) [reply modules/apps/commerce/commerce-product-test/src/testFunctional/paths/CommerceEntry.path]: request done (add curly brackets on locator variables) @lorenzolapescara

**Review by lorenzolapescara — APPROVED** [review]

**Comment by lorenzolapescara** [comment]
ci:test:sf

**Comment by lorenzolapescara** [comment]
ci:test:commerce-functional

**Comment by lorenzolapescara** [comment]
ci:forward

**Comment by fmarcio** [comment]
new PR sent https://github.com/liferay-commerce/liferay-portal/pull/1138

**Pull request closed without merging**